.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,50 +1,36 @@
-<!--- Before submitting your pull request, --->
-<!--- please complete as much as practicable of the following checklist: --->
+<!--- Type an x in the box [x] before submitting --->
 
-### Pull Request Checklist
+### Submission Checklist
 
-* [ ] Read and followed this repo's [Contributing Guidelines](https://github.com/spyder-ide/spyder/blob/master/CONTRIBUTING.md)
-* [ ] Based this PR on the latest version of the correct branch (master or 3.x)
-* [ ] Followed [PEP 8](https://www.python.org/dev/peps/pep-0008/) code style
-* [ ] Ensured this pull request hasn't eliminated unrelated blank lines/spaces,
-      modified the ``spyder/defaults`` directory, or added new icons/assets
-* [ ] Wrote at least one-line docstrings following PEP 257for any new functions
-* [ ] Added at least one unit test covering the changes, if at all possible
-* [ ] Described the changes and the motivation for them below
-* [ ] Noted what issue(s) this pull request resolves, creating one if needed
-* [ ] Included a screenshot, if this PR makes any visible changes to the UI
+<!--- I have read the Contributing Guidelines:                         --->
+<!--- https://github.com/spyder-ide/spyder/blob/master/CONTRIBUTING.md --->
+<!--- and followed PEP 8, PEP 257 and Spyder's code style:             --->
+<!--- https://github.com/spyder-ide/spyder/wiki/Dev:-Coding-Style      --->
 
+By submitting this Pull Request or typing my (user)name below,
```

Comment:
Thanks for the explanation about this.

Since we are stuck with this, can we at least move it to the bottom of the template so that this is not the first thing we see when browsing the PRs afterwards.

Reply:
Okay, I did so. There is a greater risk that people miss it though...

Reply:
Maybe. It's closer to the button to create the pull request though. Worst thing that can happens is that we will have to remind people to sign it from time to time. I really think that putting technical stuff to the front of the PR is good move in the long run for the development.

Reply:
:+1:

Reply:
Maybe you can "move" the affirmation section to the "##" level instead of "###" if you think that can help to make it more visible. Your call.

Reply:
Looks like its too late now, but good thinking.

Reply:
Sorry, @jnsebgosselin approved, so I merged.

Reply:
I thought after this afterwards. Sorry for that.

Reply:
No worries, happens to me too!